### ISSUE TITLE
Timer Edit Button Suggestions/Bugs

### ISSUE THREAD

**Issue by ghost** [GrooveOtter/grooveotter-client#26]
The dilemna is that I was uncertain how to edit the timer. Only after looking at the wires did I realize that I needed to click/double click the timer.

Suggestions
1. Always have an edit button visible(rather than hover-state in the wires) for editing the time so that the user does not have issues finding it.

Bugs
1. The timer required a double-click(sometimes more) to go into edit mode, the comps needed only a single click.
2. I do not know how to leave the timer edit mode without making changes. 
3. Circle missing after entering edit timer.
4. Edit timer should have min included.
5. Edit timer should not have arrows for increasing/decreasing time.
6. After entering a number in timer box and clicking Start task does not display the timer clock.

Questions
1. Do I make note(or add To-do tasks) of every little difference compared to wires?
2. There are differences between wires and the demo, such as the wording of Reset Timer, which is correct?

**Comment by L8D** [comment]
Unfortunately, I'm already very aware of all of these bugs. :confused:

To explain:
1. I was told the behavior is double-click (the mockups have single-click because invision doesn't support double-click).
2. I am unaware of how the input should be handled in that way. I have it set to when you hit enter or click on of the the preset options.
3. Haven't gotten to that yet.
4. I'm assuming you mean how the outside of the timer is shaded.
5. Haven't gotten to that yet.
6. That's because the input box has `type="number"` and so modern browsers put arrows in there
7. Is a legitimate bug that I forgot to configure properly.

To answer your questions:
1. Yes, even if they are obvious or already known it's good to have them noted so we can track progress and don't forget them.
2. It's up to Garreth and Ryan

**Comment by L8D** [comment]
Duplicate of #20
